cmd/guacone/cmd/files.go

```diff
@@ -207,14 +207,14 @@ func validateFlags(user string, pass string, dbAddr string, realm string, keyPat
 
 func getProcessor(ctx context.Context) (func(*processor.Document) (processor.DocumentTree, error), error) {
 	return func(d *processor.Document) (processor.DocumentTree, error) {
-		return process.Process(ctx, d)
+		return process.Process(ctx, d) // nolint:wrapcheck
 	}, nil
 }
 func getIngestor(ctx context.Context) (func(processor.DocumentTree) ([]assembler.Graph, error), error) {
 	return func(doc processor.DocumentTree) ([]assembler.Graph, error) {
 		inputs, err := parser.ParseDocumentTree(ctx, doc)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("unable to parse document tree: %v", err)
```

Comment:
probably should be %w for wrapping error

Reply:
Good idea, I forgot about that. I have changed all the `%v`’s that I have added to `%w`’s.